### DISCUSSION TITLE
Using SCI to measure purchased software

### DISCUSSION THREAD

**Discussion by seebach** [Green-Software-Foundation/sci#253] [General]
Hi,

We have made a comparison between an on-premise system and a cloud system. They are both used for the same purpose but are from different vendors. The onprem runs on 6 servers with 12 cores on each. And the cloud system auto-scales from 4 to 32 CPU's. Just from that knowledge you would expect the cloud to perform better. But the fact is that CPU consumption on the 72 cores onprem is so low it outperforms the cloud on in the SCI score. 

From these calculations, we don't see that the SCI methods provides any incentive to right-size the hardware both onprem and cloud. The auto scale feature in the cloud also does not make much of a difference when using 4 or 8 CPU's. As the average consumption of the solution is in general low with many high peaks.

Because of this we only see the SCI method only works on software that you develop and is not really usable on the software you purchase. From an optimization standpoint, there should be a penalty for having unused resources in a system.

**Comment by atg-abhishek** [comment]
Thanks for sharing this @seebach - could you share calculations that you ran for both on-prem and cloud infrastructure so that the WG can evaluate? Thanks!

**Reply by seebach** [reply]
Sure, we will do that. Could you comment on the fact that unused core doesn't change the calculation?

**Comment by Henry-WattTime** [comment]
@seebach, we would love to have you join the working group to discuss this in more depth. Very interesting question. It would be great to talk through the specific situation you find yourself in. Gadhu will reach out to coordinate.
